movies-app/src/components/MovieModal/MovieModal.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { useContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import { I18N } from '../../core/i18n';
+import { LocaleContext } from '../../core/i18n/locale';
+import './MovieModal.scss';
+import { ReactPortal } from '../ReactPortal';
+import CloseIcon from '../../core/constants/close-icon.svg';
+import genreMocks from '../../mocks/genreTypes.json';
+
+export const MovieModal = ({ closeModal, header }) => {
+  const locale = useContext(LocaleContext);
+
+  const [isSelectOptionsOpened, setIsSelectOptionsOpened] = useState(false);
+
+  const selectOptions = genreMocks.map((item) => {
+    return (
+      <label>
+        <input type="checkbox" />
+        {item}
+      </label>
+    );
+  });
+
+  return (
+    <ReactPortal>
+      <div className="movie-modal">
```

Comment:
Ok, this time the suggestion is not optional :)
Consider breaking down large JSX into smaller functions inside/outsider of component's body.
It not only improves overall readability but also opens up opportunities to create reusable pieces of code (like, form groups and fields)